src/vivarium_cluster_tools/cli.py

```diff
@@ -26,7 +26,8 @@ def psimulate():
               help='The estimated maximum memory usage in GB of an individual simulate job. The simulations will be '
                    'run with this as a limit.')
 @click.option('--pdb', 'with_debugger', is_flag=True, help='Drop into python debugger if an error occurs.')
-def run(simulation_configuration, branch_configuration, result_directory, project, peak_memory, with_debugger):
+@click.option('--redis', type=int, default=-1)
```

Comment:
without you in the same room to remind me I don't remember if this specifies number of redis instances or number of jobs to assign to an instance. Could you add a help string?